extras/dfhack_extras.init

```diff
@@ -0,0 +1,21 @@
+## Extra tools, tweaks, and UI mods - additional to standard "dfhack.init".
+
+# keybindings for Lethosor's scripts
+keybinding add Alt-S@title settings-manager
+keybinding add Alt-S@dwarfmode/Default settings-manager
+keybinding add Alt-Q@jobmanagement manager-quantity
```

Comment:
The `manager-quantity` script was replaced by `tweak manager-quantity`, which was then removed in 0.43.01 since there's no longer a limit of 30 in vanilla DF.